blackbox/docs/configuration.txt

```diff
@@ -562,6 +562,54 @@ Elasticsearch HTTP REST API
     Manipulating your data via elasticsearch API and not via SQL
     might result in inconsistent data. You have been warned!
 
+Cross-Origin Resource Sharing (CORS)
+------------------------------------
+
+Many browsers support the `same-origin policy`_ which requires web applications
+to explicitly allow requests across origins. The `cross-origin resource sharing`_
+settings in CrateDB allow for configuring these.
+
+**http.cors.enabled**
+  | *Default:* ``false``
+  | *Runtime:* ``no``
+
+  Enable or disable `cross-origin resource sharing`_.
+
+**http.cors.allow-origin**
+  | *Default:* ``<empty>``
+  | *Runtime:* ``no``
+
+  Define allowed origins of a request. ``*`` allows *any* origin (which can be
+  a substantial security risk) and by prepending a ``/`` the string will be
+  treated as a regular expression. For example ``/https?:\/\/crate.io/`` will
+  allow requests from ``http://crate.io`` and ``https://crate.io``. This setting
+  disallows any origin by default.
+
+**http.cors.max-age**
+  | *Default:* ``1728000`` (20 days)
+  | *Runtime:* ``no``
+
+  Max cache age of a preflight request in seconds.
+
+**http.cors.allow-methods**
+  | *Default:* ``OPTIONS, HEAD, GET, POST, PUT, DELETE``
+  | *Runtime:* ``no``
+
+  Allowed HTTP methods.
+
+**http.cors.allow-headers**
+  | *Default:* ``X-Requested-With, Content-Type, Content-Length``
+  | *Runtime:* ``no``
+
+  Allowed HTTP headers.
+
+**http.cors.allow-credentials**
+  | *Default:* ``false``
+  | *Runtime:* ``no``
+
+  Return the ``Access-Control-Allow-Credentials`` header.
```

Comment:
I would rephrase it somehow to say that it enables the return of the this header